boards/arm/hexiwear_k64/hexiwear_k64.dts

```diff
@@ -37,9 +37,7 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,console = &uart0;
-#ifdef CONFIG_BT
```

Comment:
The changes here are fine, but the commit message doesn't make sense anymore. Will you please update the commit message?